.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+lib
```

Comment:
`lib` sounds really general, if it's being used for the typescript compilation, should we rename it to `build` or `dist`? honestly not sure what has been adopted by the community

Reply:
will rename to `build`